### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by JesusWisord**
Adding Scrapper for:

GamePlanet
Sears
MIxUp

Features:

Pagination for Sears

Missing:

Pagination for GamePlanet and MixUp

### CODE REVIEW

File: scraper/utils/spiders.py [hunk 1]
```diff
@@ -135,5 +136,203 @@ def parse(self, response):
...
+        description = self.driver.find_elements_by_xpath(desc_xp)[0].get_attribute('innerText')
+
+        price_xp = f'//div[contains(@class, "{GamePl.PRICE_CLASS.value}")]//span'
+        price = self.driver.find_elements_by_xpath(price_xp)[0].get_attribute('innerText')
+
+        image_xp = (f'//img[@id = "{GamePl.IMAGE_ID.value}"]')
+        image_url = self.driver.find_elements_by_xpath(image_xp)[0].get_attribute('src')
+        image = image_url
+
+        yield {
```
Comment (jcalvarezj) [inline scraper/utils/spiders.py:186]: Please add the other missing fields for the records: **id_type_product** and **id_ecommerce**

File: scraper/utils/spiders.py [hunk 2]
```diff
@@ -135,5 +136,203 @@ def parse(self, response):
...
+        description = description.strip()
+
+        price_xp = f'//span[contains(@class, "{MU.PRICE_CLASS.value}")]/text()'
+        price = response.xpath(price_xp)[1].get()
+        price = price.strip()          
+
+        image_xp = (f'//img[@id="{MU.IMAGE_ID.value}"]/@src')
+        image = response.urljoin(response.xpath(image_xp).get())
+
+        yield {
```
Comment (jcalvarezj) [inline scraper/utils/spiders.py:250]: Please add the other missing fields for the records: **id_type_product** and **id_ecommerce**

File: scraper/utils/spiders.py [hunk 3]
```diff
@@ -135,5 +136,203 @@ def parse(self, response):
...
+        desc_xp = f'//div[@class = "{SEA.DESC_CLASS.value} yotpo-main-widget"]/@data-description'
+        description = response.xpath(desc_xp)[0].get()
+
+        price_xp = f'//p[@class = "{SEA.PRICE_CLASS.value}"]/text()'
+        price = response.xpath(price_xp).get()
+
+        image_xp = (f'//ul[contains(@class,{SEA.IMAGE_CLASS.value})]/li/img/@src')
+        image = response.urljoin(response.xpath(image_xp)[0].get())
+
+        yield {
```
Comment (jcalvarezj) [inline scraper/utils/spiders.py:310]: Please add the other missing fields for the records: **id_type_product** and **id_ecommerce**

File: scraper/utils/spiders.py [hunk 4]
```diff
@@ -44,7 +46,7 @@ def parse_product(self, response):
 
         image_xp = (f'//div[contains(@class, "{OLX.IMG_DIV_CLASS.value}")]//'
                     'img/@src')
-        image = response.urljoin(response.xpath(image_xp).get())
+        image = response.xpath(image_xp).get()
```
Comment (jcalvarezj) [inline scraper/utils/spiders.py:49]: Is this change necessary? The absolute path/url stops working for me if I change this to **xpath** instead of **urljoin**

**Comment by jcalvarezj** [comment]
Ah, @JesusWisord , please also remember to add the new sites to the documentation of  **scrapy.py**'s **run** method as part of your changes

**Comment by JesusWisord** [comment]
Changes done in branch dev
